samples/nrf9160/lwm2m_client/overlay-lwm2m-1.1.conf

```diff
@@ -1,5 +1,3 @@
-CONFIG_BASE64=y
 CONFIG_LWM2M_VERSION_1_1=y
-CONFIG_LWM2M_RW_SENML_JSON_SUPPORT=y
-CONFIG_LWM2M_COAP_MAX_MSG_SIZE=1024
+CONFIG_LWM2M_COAP_MAX_MSG_SIZE=1280
 CONFIG_LWM2M_COMPOSITE_PATH_LIST_SIZE=10
```

Comment:
Why do you touch this file in this commit?

Reply:
I can remove these changes.